docs/testing/testing-with-puppeteer.md

```diff
@@ -82,6 +82,16 @@ integration](../testing/continuous-integration.md):
   Integration (CI) services because small races are amplified in those
   environments; this often explains failures in CI that cannot be
   easily reproduced locally.
+* Does the test fail when you are typing (filling the form) on modal? If so,
+  the problem is likely that the modal was focused after it starts typing.
+  Puppeteer starts typing after focusing on the text field, sending keystrokes
+  one after another. So, calling `.focus` on modal after it starts
+  typing shifts focus from the text field resulting in partially typed data.
+  A very simple fix for this is to wait until the modal is focused after
```

Comment:
"A very simple fix for this ..." => "This recommended fix for this is to wait until the modal is focused before starting typing"

Reply:
You meant, "~This~ The recommended fix for this is to wait until the modal is focused before starting typing"?